relay-event-normalization/src/replay.rs

```diff
@@ -60,15 +62,22 @@ pub fn validate(replay: &Replay) -> Result<(), ReplayError> {
 
 /// Adds default fields and normalizes all values in to their standard representation.
 pub fn normalize(
-    replay: &mut Replay,
+    replay: &mut Annotated<Replay>,
```

Comment:
We may want to follow the same structure for replay normalization as events, but that's out of scope for now.